buildenv/docker/test/Dockerfile

```diff
@@ -65,7 +65,7 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 # Install Docker module to run test framework
-RUN echo yes | cpan install JSON Text::CSV
+RUN echo yes | cpan install JSON Text::CSV XML::Parser
```

Comment:
I think we're also missing `curl` in the above `apt-get` command. I just built the Docker image and it complained that it is not installed. We may want to address this here given we're fixing the XML::Parser dependency.

Reply:
We can likely let the tests complete and make note of them here because adding a new change will erase records of the testing from the PR.

Reply:
In the previous attempt at this pull request, the discussion around whether we should be keeping this Dockerfile or deleting them was mentioned (discussion in https://github.com/eclipse/openj9/issues/5276).  Also note that the dependency on XML::Parser was removed in a cautious manner for the z/OS platform only to enable z/OS testing.  

As discussed already, post-July release we will revisit the prototype Java rewrite for next phase of removing dependencies.

Reply:
Should we still add `curl` and XML::Parser to the current Dockerfile? I'm fine either way as I'm aware of the issue. Was hoping to save some time for others who may try to use the buildenv/docker/test/Dockerfile as it's currently missing prerequisites noted reference via the _Reproducing Test Failures Locally_ wiki article [1].

[1] https://github.com/eclipse/openj9/blob/master/test/docs/Prerequisites.md